packages/wp-now/src/wp-now.ts

```diff
@@ -213,17 +210,21 @@ export default class WPNow {
 	}
 
 	static #inferMode(projectPath: string): Exclude<WPNowMode, 'auto'> {
-		const hasIndexPhp = fs.existsSync(path.join(projectPath, 'index.php'));
-		const hasWpContentFolder = fs.existsSync(
-			path.join(projectPath, 'wp-content')
-		);
+		const fileExists = (filename) =>
+			fs.existsSync(path.join(projectPath, filename));
 
+		if (
+			fileExists('wp-content') &&
```

Comment:
If we let go of modes and always mounted whatever's available, we'd just mount each `wp-*` directory here and get the `wp-content` mode for free.